src/widgets/QDisassemblyView.cpp

```diff
@@ -1283,12 +1283,35 @@ void QDisassemblyView::drawJumpArrows(QPainter &painter, const DrawingContext *c
 		return a.distance < b.distance;
 	});
 
+	auto isLineOverlap = [&](int line1_head, int line1_tail, int line2_head, int line2_tail, bool edge_overlap = true) -> bool {
```

Comment:
You are not using this default `= true`, so no point in it.

Reply:
fixed